libs/platform/src/lib/components/form/combobox/commons/base-combobox.ts

```diff
@@ -75,6 +76,13 @@ export abstract class BaseCombobox extends CollectionBaseInput implements AfterV
     @Input()
     maxHeight = '250px';
 
+    /**
+     *  The state of the form control - applies css classes.
+     *  Can be `success`, `error`, `warning`, `information` or blank for default.
+     */
+    @Input()
+    state: FormStates;
```

Comment:
Let's add default `state` value

Reply:
As you can see, you should skip this property for using default state.

Reply:
Thanks, got it
@fkolar could you please review it?